cterasdk/core/users.py

```diff
@@ -13,6 +15,24 @@ class Users(BaseCommand):
 
     _default_fields = ['name']
 
+    def get(self, user_account, include=None):
+        """
+        Get a user account
+
+        :patam UserAccount user_account: User account, including the user directory and user name
+        :param list[str] include: List of fields to retrieve, defaults to ['name']
+        :return: The user account, including the requested fields
+        """
+        directory, user = user_account
+        baseurl = ('/users/' + user) if directory == 'local' else '/domains/' + directory + '/adUsers/' + user
+        include = union.union(include or [], Users._default_fields)
+        param = query.QueryParamBuilder().include(include).build()
+        try:
+            user = query.query(self._portal, baseurl, param)[1]
```

Comment:
Why [1] and not [0], if it is in fact correct, please add a comment since it is not intuitive. In addition, no need for the "user" variable, just return what you get from the query

Reply:
Revised the code to use **get_multi** instead of **query**